runtime/src/bank.rs

```diff
@@ -6709,6 +6716,47 @@ impl Bank {
         if new_feature_activations.contains(&feature_set::update_hashes_per_tick6::id()) {
             self.apply_updated_hashes_per_tick(UPDATED_HASHES_PER_TICK6);
         }
+
+        if new_feature_activations.contains(&feature_set::accounts_lt_hash::id()) {
+            // Activating the accounts lt hash feature means we need to have an accounts lt hash
+            // value at the end of this if-block.  If the cli arg has been used, that means we
+            // already have an accounts lt hash and do not need to recalculate it.
+            if self
+                .rc
+                .accounts
+                .accounts_db
+                .is_experimental_accumulator_hash_enabled()
+            {
+                // We already have an accounts lt hash value, so no need to recalculate it.
+                // Nothing else to do here.
+            } else {
+                info!(
+                    "Calculating the accounts lt hash as part of feature activation; \
+                    this may take some time...",
+                );
+                // We must calculate the accounts lt hash now as part of feature activation.
+                // Note, this bank is *not* frozen yet, which means it will later call
+                // `update_accounts_lt_hash()`.  Therefore, we calculate the accounts lt hash based
+                // on *our parent*, not us!
+                let parent_ancestors = {
+                    let mut ancestors = self.ancestors.clone();
+                    ancestors.remove(&self.slot());
+                    ancestors
+                };
+                let parent_slot = self.parent_slot;
+                let (parent_accounts_lt_hash, duration) = meas_dur!({
+                    self.rc
+                        .accounts
+                        .accounts_db
+                        .calculate_accounts_lt_hash_at_startup_from_index(
+                            &parent_ancestors,
+                            parent_slot,
+                        )
+                });
+                *self.accounts_lt_hash.get_mut().unwrap() = parent_accounts_lt_hash;
```

Comment:
self.accounts_lt_hash() should be all zeros here, right? 
Can we assert this?

Reply:
Then imaging that a  validator downloaded a remote snapshot, which was generated by a node that enabled lt hash.
Will this node start to compute lt hash, even though it didn't pass the cli arg?

Reply:
Then will each bank carries the lt-hash from the old snapshot? 
When this node make a snapshot, will it put the out dated hash into the snapshot?

Reply:
If some other node download a snapshot from this node, will it use the wrong lt-hash if this other node has the lt-hash enabled in its cli?

Reply:
ok. then we are good.
👍